sdk/src/Core/Amazon.Runtime/Pipeline/Contexts.cs

```diff
@@ -34,6 +34,7 @@ public interface IRequestContext
         RequestMetrics Metrics { get; }
         ISigner Signer { get; set; }
         BaseIdentity Identity { get; set; }
+        AWSCredentials DefaultAWSCredentials { get; }
```

Comment:
Same comment as the property name for service clients to use the name `ExplicitAWSCredentials` instead of reusing `DefaultAWSCredentials`.

Reply:
Updated in https://github.com/aws/aws-sdk-net/pull/3958/commits/d3c3771378f6dca00ecaee25c18708817c1e07a1